src/dotnet/devcontainer-template.json

```diff
@@ -11,14 +11,18 @@
             "type": "string",
             "description": ".NET version:",
             "proposals": [
+                "7.0",
                 "6.0",
                 "3.1",
+                "7.0-bullseye",
                 "6.0-bullseye",
                 "3.1-bullseye",
+                "7.0-jammy",
+                "6.0-jammy",
                 "6.0-focal",
                 "3.1-focal"
             ],
-            "default": "6.0-bullseye"
+            "default": "7.0-bullseye"
```

Comment:
Can we simplify this to just `7.0`? I feel like the default value should be very simple with the more specific distro choice available as an option. Most users probably care more about the .NET SDK/runtime version and not as much about the underlying distro version they are developing on (and if they do care, they can select a more specific option).

Reply:
Makes sense. Updated